services/static-webserver/client/source/class/osparc/dashboard/NewPlusMenu.js

```diff
@@ -15,6 +15,45 @@
 
 ************************************************************************ */
 
+/**
```

Comment:
I guess this can go away if not used

Reply:
Here some tools to help you

https://domsignal.com/json-schema-generator

Reply:
It might come in a follow up PR 👍